lib/model/indexhandler.go

```diff
@@ -489,14 +489,8 @@ func prepareFileInfoForIndex(f protocol.FileInfo) protocol.FileInfo {
 	if f.IsReceiveOnlyChanged() {
 		f.Version = protocol.Vector{}
 	}
-	// The trailer with the encrypted fileinfo is device local, don't send info
-	// about that to remotes
+	// The trailer with the encrypted fileinfo is device local, announce the size without it to remotes.
```

Comment:
I wonder if this subtraction should then really happen in ToWire() instead

Reply:
In principle fully agreed, also the receive-only vector reset. However for both of these I am somewhat confused about their exact purpose/effect, so I chickened out and just didn't touch them.